externals/kyuubi-trino-engine/bin/trino-engine.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+if [[ -z ${JAVA_HOME} ]]; then
+  echo "[ERROR] JAVA_HOME IS NOT SET! CANNOT PROCEED."
+  exit 1
+fi
+
+RUNNER="${JAVA_HOME}/bin/java"
+
+if [[ "$TRINO_ENGINE_HOME" == "$KYUUBI_HOME/externals/engines/trino" ]]; then
+  TRINO_CLIENT_JAR="$TRINO_ENGINE_JAR"
+else
+  echo "\nTRINO_ENGINE_HOME $TRINO_ENGINE_HOME doesn't match production directory, assuming in development environment..."
+  TRINO_CLIENT_JAR=$(find "$TRINO_ENGINE_HOME/target" -regex ".*/kyuubi-trino-engine_.*.jar")
```

Comment:
Should we exclude `*-sources.jar` `*-tests.jar` etc. ?

Reply:
There is no `*-sources.jar`, exclude `*-tests.jar` and `*-javadoc.jar`

Reply:
`*-sources.jar` `*-javadoc.jar` should be created when `-Papache-release`, and we need create a test jar final for it module, just like flink-sql-engine

Reply:
Got it, and done.